File: CS/CustomLabelsAndStatusesExample/Model/CustomLabel.cs
#region #customlabelusings
using DevExpress.Mvvm.POCO;
using System.Windows.Media;
#endregion #customlabelusings

namespace CustomLabelsAndStatusesExample {
    #region #customlabel
    public class CustomLabel {
        public static CustomLabel Create() {
            return ViewModelSource.Create(() => new CustomLabel());
        }
        protected CustomLabel() { }
        public virtual int Id { get; set; }
        public virtual string Caption { get; set; }
        public virtual Color Color { get; set; }
    }
    #endregion #customlabel
}

File: VB/CustomLabelsAndStatusesExample/Model/CustomLabel.vb
#Region "#customlabelusings"
Imports DevExpress.Mvvm.POCO
Imports System.Windows.Media
#End Region ' #customlabelusings

Namespace CustomLabelsAndStatusesExample
    #Region "#customlabel"
    Public Class CustomLabel
        Public Shared Function Create() As CustomLabel
            Return ViewModelSource.Create(Function() New CustomLabel())
        End Function
        Protected Sub New()
        End Sub
        Public Overridable Property Id() As Integer
        Public Overridable Property Caption() As String
        Public Overridable Property Color() As Color
    End Class
    #End Region ' #customlabel
End Namespace
